CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to Project
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to Project and its packages, which are hosted in the [Link](https://github.com/dhruvil009/HW01) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Project and Packages](#project-and-packages)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [NCSU Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior.
+
+## What should I know before I get started?
+
+### Project and Packages
+The project is yet to be decided and we will update on project description soon. We will also update on packages and requirements once the project has been finalized
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for Project. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
```

Comment:
Should I create a project for this repository?

Reply:
Not needed now, we can discuss later on project and dashboard